cmd/tailscale/cli/web.go

```diff
@@ -76,7 +78,8 @@ func runWeb(ctx context.Context, args []string) error {
 		return fmt.Errorf("too many non-flag arguments: %q", args)
 	}
 
-	webHandler := http.HandlerFunc(web.Handle)
+	webServer := web.NewServer(webArgs.dev, nil)
+	webHandler := http.HandlerFunc(webServer.ServeHTTP)
```

Comment:
there's no need for `webHandler` at all now.  You can just pass webServer directly wherever you need an `http.Handler` below.  e.g.

```go
cgi.Serve(webServer)
```